scripts/components/item_component/item_component.gd

```diff
@@ -20,6 +20,10 @@ func _ready() -> void:
 
 
 func _on_interaction_component_interacted() -> void:
-	Events.picked_up_item.emit(item)
+	# TODO : add use item in item manager instead?
```

Comment:
I made a Recipe that combines Blacklight and battery, so we don't need to modify this.

We should not change the item_component script. If we want to do custom interaction we should instead hook the interacting event to a custom script function.